public/static/locales/en/common.json

```diff
@@ -0,0 +1,29 @@
+{
+    "admin": "Admin",
+    "all": "All",
+    "analyses": "Analyses",
+    "apps": "Apps",
+    "cyverse": "CyVerse",
+    "dashboard": "Dashboard",
+    "data": "Data",
+    "deTitle": "Discovery Environment",
+    "discovery": "<b>D</b>iscovery",
+    "drawerMenuAriaLabel": "drawer menu",
+    "environment": "<b>E</b>nvironment",
+    "intercomAriaLabel": "Chat with CyVerse support",
+    "login": "Login",
+    "logout": "Logout, {{name}}",
+    "more": "More",
+    "newNotificationAriaLabel": "Show New Notifications",
+    "openDrawer": "Open Drawer",
+    "search": "Search...",
+    "searchAriaLabel": "search",
+    "searchButtonAriaControl": "search button",
+    "searchFilterAriaControl": "search filter menu",
+    "searchFilterAriaLabel": "search filter",
+    "searchFilterMenuAriaLabel": "select search filter",
+    "settings": "Settings",
+    "userMenuAriaLabel": "user menu",
+    "vice": "VICE",
+    "viewOutput": "View output"
+}
```

Comment:
This is the translation file. This can be broken up into several namespaces like data, apps etc...